### ISSUE TITLE
Errors with Selenium 4.0.0.b3 

### ISSUE THREAD

**Issue by xWilliamsx** [MarketSquare/robotframework-seleniumtestability#58]
Hello,

I apologize as I am new to this, but wanted to see if there is a work around for this.

When using Selenium Library version 4.0.0b3 I am getting the follow error below, yet when using version 3.141.0 it will run successfully. We installed the latest version of Selenium to test against Edge browser. Do you have a fix for this?

[ ERROR ] Error in file ‘C:\GIT\Repos\Eclipse\UPEnterprise_Robot\tests\Regressions\UPExchange\UPXREGEditProperties.robot’ on line 2: Initializing library ‘SeleniumLibrary’ with arguments [ plugins=SeleniumTestability;False;5 Seconds;True ] failed: Importing test library ‘SeleniumTestability’ failed: AttributeError: type object ‘DesiredCapabilities’ has no attribute ‘PHANTOMJS’
Traceback (most recent call last):
File “C:\AppData\Local\Programs\Python\Python37\lib\site-packages\SeleniumTestability_init_.py”, line 2, in
from .plugin import SeleniumTestability
File “C:\AppData\Local\Programs\Python\Python37\lib\site-packages\SeleniumTestability\plugin.py”, line 43, in
class SeleniumTestability(LibraryComponent):
File “C:\AppData\Local\Programs\Python\Python37\lib\site-packages\SeleniumTestability\plugin.py”, line 148, in SeleniumTestability
“phantomjs”: DesiredCapabilities.PHANTOMJS,
PYTHONPATH:
C:\GIT\Repos\Eclipse\UPEnterprise_Robot
C:\AppData\Local\Programs\Python\Python37\python37.zip
C:\AppData\Local\Programs\Python\Python37\DLLs
C:\AppData\Local\Programs\Python\Python37\lib
C:\AppData\Local\Programs\Python\Python37
C:\AppData\Roaming\Python\Python37\site-packages
C:\AppData\Local\Programs\Python\Python37\lib\site-packages

## Expected Behavior
Drag and drop works correctly

## Actual Behavior
Test throw errors right at the beginning and when it reaches drag and drop it fails the test

## Steps to Reproduce the Problem

  1. Upgrade to 4.0.0b3
  2. Run any test that has seleniumtestability
  3. Run test with drag and drop

robotframework==4.0.1
robotframework-pythonlibcore==2.2.1
robotframework-requests==0.8.0
robotframework-selenium2library==3.0.0
robotframework-seleniumlibrary==5.1.3
robotframework-seleniumtestability==1.1.0
selenium==3.141.0

**Comment by rasjani** [comment]
This is because phantomjs has been deprecated in selenium4..

Not really a bug but will require some changes to ST ..

**Comment by xWilliamsx** [comment]
Thank you for the very prompt response! I agree not really an error I apologize I'm still pretty new to this and wasn't sure how to reach out. Do you have an estimate for an update? If not that's fine as well.

Thank you again for your time!

Sent from my T-Mobile 5G Device
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by xWilliamsx** [comment]
Hello, I was wondering if you had a work around until ST is updated?

**Comment by emanlove** [comment]
Update: I've been working through the libraries [acceptance tests to check my changes](https://github.com/rasjani/robotframework-seleniumtestability/pull/61). There are two failing test, both Firefox. One related to log errors I think is an issue with old jQuery version and not indicative of library issues. The other, also Firefox, it's unclear at the moment why it is occurring.

I am looking today (18 Oct 2021) but don't have a firm completion date.

**Comment by emanlove** [comment]
A Selenium 4 compatible release candidate has been made and can be installed by running

```bash
pip install --pre --upgrade robotframework-seleniumtestability
```
or using
```bash
pip install robotframework-seleniumtestability==2.0.0rc1
```

**Comment by emanlove** [comment]
This is fixed with the latest 2.0.0 release.
